validator_worker/src/follower.rs

```diff
@@ -14,6 +14,7 @@ enum InvalidNewState {
     RootHash,
     Signature,
     Transition,
+    Health,
```

Comment:
Should we add a doc comment, which states this is for the Unsignable Heath InvalidNewState

Reply:
we can rename it to UnsignableHealth if you want